### PR TITLE
troubleshooting

### PR DESCRIPTION
**Pull request by joergmschulz**
doesn't work if passwords are enforced for shared documents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,9 @@ enter the room by pointing to the arrow button
 use the ... menu
 
 select the correct BBB room and enter the room
+
+# Troubleshooting
+- before installing, make sure your BBB is running correctly
+- sameSite Cookies issue in modern chrome/chromium is to be expected. Always use the same tld.org
```
Comment (sualko) [inline README.md:41]: What do you mean with "same site cookies issue"?

Do you have a screenshot from your error message?

Comment (joergmschulz) [reply README.md:41]: see https://github.com/ramezrafla/spreed-bigbluebutton

Comment (sualko) [reply README.md:41]: What should I see? The architecture is completely different and we should not suffer from any cross domain issues.

Comment (joergmschulz) [reply README.md:41]: will retest and eventually remove this line, sorry

Comment (sualko) [reply README.md:41]: > sorry

no problem

**Comment by sualko** [comment]
I added you as collaborator. Feel free to merge on your own behalf.

File: README.md (hunk 1 above)
Comment (joergmschulz) [reply README.md]: retested & removed that line - we're not having this issue. Sorry.

**Pull request merged**